python/ray/air/util/tensor_extensions/pandas.py

```diff
@@ -1430,22 +1430,3 @@ def _is_boolean(self):
 TensorArray._add_arithmetic_ops()
 TensorArray._add_comparison_ops()
 TensorArray._add_logical_ops()
-
-
-@PublicAPI(stability="beta")
```

Comment:
I assume no one uses this, but it's technically a public API, so I'm not sure if we should keep it just to be safe.